apps/maestro/src/features/RegisterRemoteTokens/RegisterRemoteTokens.tsx

```diff
@@ -193,7 +191,8 @@ export const RegisterRemoteTokens: FC<RegisterRemoteTokensProps> = (props) => {
   const {
     writeContractAsync: registerInterchainTokensAsync,
     reset: resetInterchain,
-  } = useRegisterRemoteInterchainTokens({
+    simulationError: interchainSimulationError,
+  }: any = useRegisterRemoteInterchainTokens({
```

Comment:
I think that we can also get rid of the `any` if we add `simulationError: undefined` both in the fallback value and and [here](https://github.com/axelarnetwork/axelarjs/blob/fix-remote-deployment-readyness/apps/maestro/src/features/RegisterRemoteTokens/hooks/useRegisterRemoteInterchainTokens.ts#L105]